nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -321,31 +322,22 @@ public void testExistingTableUpdate() {
   }
 
   @Test
-  public void testCommitsOutsideOfCatalogApi() throws NessieNotFoundException, NessieConflictException {
```

Comment:
Is this set of changes required? I think we just changed this in the last PR

Reply:
yes those are required as the "reference loading" behavior is how it was previously and therefore that test that I rewrote is now valid again. This was unfortunately an oversight on my side as we should have been using a memoized supplier from the very beginning